exec.bash

```diff
@@ -32,6 +32,7 @@ fi
 if [ "$EXECMODE" == "base" ]; then
     # DOCKER_REGISTRY and WORKSPACE_DEVEL_IMAGE from settings.bash
     IMAGE_NAME=${BASE_REGISTRY:+${BASE_REGISTRY}/}$WORKSPACE_BASE_IMAGE
+    [ "$ARCH" != "x86_64" ] && export IMAGE_NAME=$(dirname $IMAGE_NAME)/$ARCH/$(basename $IMAGE_NAME)
```

Comment:
I think this is not needed, as the arch should be part of $WORKSPACE_BASE_IMAGE from the settings.bash

Reply:
Mh I am not sure about it. I find it quite convenient to be able to leave the image names in the settings.bash untouched and have everything work smoothly when changing the ARCH parameter. However I agree that this might lead to some confusion. Any suggestion on how to solve this elegantly?

Reply:
hmm, then we could just disable nvidia base images when no arch is set...